scripts/src/java/org/oppia/android/scripts/coverage/CoverageReporter.kt

```diff
@@ -9,6 +9,63 @@ import java.io.File
 /** Minimum coverage percentage required. */
 const val MIN_THRESHOLD = 70
 
+/** ANSI escape codes for colors. */
```

Comment:
```suggestion
/* ANSI escape codes for colors. */

```

This isn't a KDoc (since it has no element to document.

Reply:
Done, updated as suggested.